task/sast-shell-check-oci-ta/0.2/MIGRATION.md

```diff
@@ -0,0 +1,4 @@
+# Migration from 0.1 to 0.2
+
+* A new, optional parameter `TARGET_DIR` has been added, which defaults to `"."`. This can be set to the name of a valid
```

Comment:
If the default behavior is compatible with the previous version of the task, do we need to create a new version of the task and bother end users with the migration?

Reply:
D'oh! I should have read the readme closer, at the ends of the Versioning sections it says:

> Adding a new parameter with a default value does not require a task version increase.

Fixed now.